designs/charlie/src/front.mjs

```diff
@@ -264,7 +295,17 @@ function draftCharlieFront({
         )
         .close()
         .trim()
-        .attr('class', 'fabric sa')
+        .setClass('fabric sa')
+
+      // Draw the right leg fly extension (not for dolls)
+      if (measurements.waist > 500) {
+        let FlyRightLegExtensionSa = paths.flyRightLegExtension.offset(sa)
```

Comment:
Unless you plan to overwrite the variable later on, we prefer to use `const` in our codebase.
It's more declarative, somewhat more efficient, and reduces cognitive overhead when reading code.

Unless I missed something (I'm reading on my phone) this could be a `const`